data/data/bootstrap/files/usr/local/bin/bootkube.sh.template

```diff
@@ -50,6 +51,21 @@ then
 	cp auth/kubeconfig /etc/kubernetes/kubeconfig
 fi
 
+if [ ! -d config-bootstrap ]
+then
+	echo "Rendering config core manifests..."
```

Comment:
Can we replace "config core" with "config.openshift.io"?  So many things are "core", I expect "config.openshift.io"  will be harder to misunderstand.

Reply:
"cluster config manifests" like the operator? Calling the same thing with different names is not a good idea.